system/mongodb/mongodb.SlackBuild

```diff
@@ -101,6 +101,9 @@ find -L . \
 # thanks gentoo, also for these patches
 for i in $CWD/patches/* ; do patch -p1 < $i ; done
 
```

Comment:
I think we should use the misc/rmlint/rmlint.SlackBuild scons handling as an example

We can pull the job counter from MAKEFLAGS and this should be many time faster building.

Reply:
Done